src/modules/comment.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] it would be better to remove the `Eslint disable ` and the previous reviewer asked because Eslint ensures you write clean and pro code 
try to use camelcase like `ThisOne` not snake_case like `this_one`

Reply:
Hi @youmari, Thank for you pointing this out but trust me we tried removing it but when we do, we get an error and this messed up the code. The camelcase was how the data was stored in the API so its difficult to remove it. you can check for yourself - https://ll.thespacedevs.com/2.2.0/launch/upcoming/?mode=list&status=8&ordering=window_end